scripts/scripts.js

```diff
@@ -33,6 +35,59 @@ function buildHeroBlock(main) {
   }
 }
 
+function buildAppDowloadBlock(main) {
+  const appDownloadSection = main.querySelector('.section.app-download');
```

Comment:
We shouldn't be dealing with section level styles or elements in block handlers.